@here/harp-mapview/lib/DataSource.ts

```diff
@@ -195,6 +196,18 @@ export abstract class DataSource extends THREE.EventDispatcher {
      */
     abstract getTilingScheme(): TilingScheme;
 
+    /**
+     * Returns a [[ElevationRangeSource]] so that [[MapView]] can compute proper bounding boxes that
+     * take the elevation range of the data into account.
+     * There can be only one [[ElevationRangeSource]] per [[MapView]]. [[MapView]] will take the
```

Comment:
> There can be only one [[ElevationRangeSource]] per [[MapView]]. (...)

Why? That means that we shouldn't even add `ElevationRangeSource` to DataSource interface, but add it to `MapView` ... (at least at this stage).

On the other hand, at this stage, MapView doesn't even use `ElevationRangeSource` and i see no reason why two separate DS would have two different elevation datasources working properly with two separate `ElevationRangeSource`s supporting different tiling schemes.